File: app/src/main/java/ru/cities/game/MainActivity.java
package ru.cities.game;

import android.os.Bundle;

import androidx.fragment.app.FragmentActivity;

import ru.cities.game.fragment.MainFragment;

public class MainActivity extends FragmentActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        setTheme(R.style.Theme_GameCities);
        super.onCreate(savedInstanceState);
        setContentView(R.layout.main_activity);
        if (savedInstanceState == null)
            getSupportFragmentManager().beginTransaction().add(R.id.container, MainFragment.class, null).commit();
    }
}

File: app/src/main/java/ru/cities/game/MainActivity.kt
package ru.cities.game

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import ru.cities.game.fragment.MainFragment

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        setTheme(R.style.Theme_GameCities)
        super.onCreate(savedInstanceState)
        setContentView(R.layout.main_activity)
        if (savedInstanceState == null)
            supportFragmentManager.beginTransaction().add(R.id.container, MainFragment()).commit()
    }
}
